CefSharp/ISelectCertCallback.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright © 2010-2016 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+using System;
+using System.Security.Cryptography.X509Certificates;
+
+namespace CefSharp
+{
+    /// <summary>
+    /// Callback interface used for fetching the selected certificate.
+    /// </summary>
+    public interface ISelectCertCallback : IDisposable
```

Comment:
This should be `ISelectClientCertificateCallback`. Keep the existing convention so it's easier to map the `CEF` methods to `CefSharp` ones.